README.md

```diff
@@ -1,72 +1,204 @@
-# Microsoft Azure SDK for Go
-[![GoDoc](https://godoc.org/github.com/Azure/azure-sdk-for-go?status.svg)](https://godoc.org/github.com/Azure/azure-sdk-for-go) 
+// azure-sdk-for-go provides Go packages for using Azure services
+
+# Azure SDK for Go
+
+[![godoc](https://godoc.org/github.com/Azure/azure-sdk-for-go?status.svg)](https://godoc.org/github.com/Azure/azure-sdk-for-go) 
 [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-go.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-go) 
 [![Go Report Card](https://goreportcard.com/badge/github.com/Azure/azure-sdk-for-go)](https://goreportcard.com/report/github.com/Azure/azure-sdk-for-go)
 
-This is Microsoft Azure's core repository for hosting Go packages which offer a more convenient way of targeting Azure
-REST endpoints. Here, you'll find a mix of code generated by [Autorest](https://github.com/Azure/autorest) and hand
-maintained packages.
+:exclamation: **NOTE:** This project is in preview and breaking changes are 
+introduced frequently. Therefore, vendoring dependencies is even more
+important than usual. We use [dep](https://github.com/golang/dep).
 
-> **NOTE:** This repository is under heavy ongoing development and should be considered a preview. Vendoring your
-dependencies is always a good idea, but it is doubly important if you're consuming this library.
+To be notified about updates and changes, subscribe to the [Azure update feed][].
 
-# Installation
-- If you don't already have it, install [the Go Programming Language](https://golang.org/dl/).
-- Go get the SDK:
+**Install:**
 
-```
+```sh
 $ go get -u github.com/Azure/azure-sdk-for-go/...
 ```
 
-> **IMPORTANT:** We highly suggest vendoring Azure SDK for Go as a dependency. For vendoring dependencies, Azure SDK
-for Go uses [dep](https://github.com/golang/dep).
+**Use:**
+
+1. Import a package from the [services][1] directory. See [SERVICES.md][] for a list.
+1. Create and authenticate a client with a `New*Client` func, e.g.
+   `c := compute.NewVirtualMachinesClient(...)`.
+1. Invoke API methods using the client, e.g. `c.CreateOrUpdate(...)`.
+1. Handle responses.
+
+[1]: https://github.com/Azure/azure-sdk-for-go/tree/master/services
+
+For example, to create a new VM (substitute your own values for angle brackets):
```

Comment:
I decided to check and no other SDK has an inline example like this. They instead link out to the samples directory, which is what we should do here as well to:

1. Direct users to where they can actually find samples
2. Avoid drift between the SDK development and the sample code

Reply:
i think it should be shorter but i think a small example helps readers better understand the "Getting Started" steps. I replaced VM with Vnet, what do you think?
we'll have to be sure this small snippet keeps pace with code changes, good point.